Public/Sdk/SelfHost/BuildXL/Tools/QTest/Tool.QTestRunner.dsc

```diff
@@ -258,6 +258,8 @@ export interface QTestArguments extends Transformer.RunnerArguments {
     qTestAdditionalOptions?: string;
     /** Path to runsettings file that will be passed on to vstest.console.exe. */
     vstestSettingsFile?: File;
+    /** Optionally override to increase the weight of test pips that require more machine resources */
+    weight?: number;

```

Comment:
If we expect this to be truely common, perhaps it is worth adding it to Transformer.RunnerArguments?

Reply:
I was debating that. My concern was that it would be misleading if it were available to specify on everything that derives from RunnerArguments but only a small number of things implement it. I figured we could always promote if as we make it more ubiquitous in our SDK. What do you think?

Reply:
Can we foresee any other usage of the "weight" feature in the near future? If not, I think it would be fine to keep it only in the QTestArguments. We can move it to Transformer.RunnerArguments later when the second consumer other than qtest pip emerge.